From 41cc1ee0698106074d819bb2fa3672ef91c53992 Mon Sep 17 00:00:00 2001
From: KalimochoAz <calimochoazucarado@gmail.com>
Date: Sat, 1 Sep 2012 03:15:04 +0200
Subject: [PATCH 10/13] Review power freq on screen off code

Review general code
removed non sense comparation  "on ?" since is extra

Change-Id: I285368dbb449e4234e9cea8846c087fc4f3200f3
---
 power/power_tuna.c | 10 +++-------
 1 file changed, 3 insertions(+), 7 deletions(-)

diff --git a/power/power_tuna.c b/power/power_tuna.c
index 80330a1..0789e8f 100644
--- a/power/power_tuna.c
+++ b/power/power_tuna.c
@@ -143,13 +143,9 @@ static void tuna_power_set_interactive(struct power_module *module, int on)
         if (len != -1 && strncmp(buf, screen_off_max_freq,
                 strlen(screen_off_max_freq)) != 0)
             memcpy(scaling_max_freq, buf, sizeof(buf));
-
-        sysfs_write("/sys/devices/system/cpu/cpu0/cpufreq/scaling_max_freq",
-                    on ? scaling_max_freq : screen_off_max_freq);
-    }
-    else
-        sysfs_write("/sys/devices/system/cpu/cpu0/cpufreq/scaling_max_freq",
-                    on ? scaling_max_freq : screen_off_max_freq);
+        sysfs_write(SCALINGMAXFREQ_PATH, screen_off_max_freq);
+    } else
+        sysfs_write(SCALINGMAXFREQ_PATH, scaling_max_freq);
 }
 
 static void tuna_power_hint(struct power_module *module, power_hint_t hint,
-- 
1.8.1.1

